apps/desktop/src/components/ChromeHeader.svelte

```diff
@@ -124,10 +124,15 @@
 
 	.center {
 		position: absolute;
-		left: 0;
 		width: 100%;
+		left: 0;
 		display: flex;
 		justify-content: center;
+		pointer-events: none;
+
+		& * {
+			pointer-events: auto;
+		}
```

Comment:
What does this fix?

Reply:
Looked closer into this one, and it turns out it was necessary actually. 

Although the direct child of that `.center` class div is a svelte component, that component then has more children defined within this `ChromeHeader.svelte` file which do get this `& * { pointer-events: auto; }` applied to them. Specifically the `.selector-series-select` div in the `customSelectButton()` snippet.

Reply:
However, looks like this manner of aboslutely centering the element works as well. The element is then only the width of the content, not the whole viewport, and therefore doesn't need any of the `pointer-events` tomfoolery: https://github.com/gitbutlerapp/gitbutler/pull/6234